src/main/kotlin/no/nav/amt/deltaker/bff/testdata/TestdataService.kt

```diff
@@ -0,0 +1,123 @@
+package no.nav.amt.deltaker.bff.testdata
+
+import kotlinx.coroutines.delay
+import no.nav.amt.deltaker.bff.deltaker.DeltakerService
+import no.nav.amt.deltaker.bff.deltaker.PameldingService
+import no.nav.amt.deltaker.bff.deltaker.forslag.kafka.ArrangorMeldingProducer
+import no.nav.amt.deltaker.bff.deltaker.model.Deltaker
+import no.nav.amt.deltaker.bff.deltaker.model.Pamelding
+import no.nav.amt.deltaker.bff.deltaker.model.Utkast
+import no.nav.amt.deltaker.bff.deltakerliste.Deltakerliste
+import no.nav.amt.deltaker.bff.deltakerliste.DeltakerlisteService
+import no.nav.amt.deltaker.bff.deltakerliste.tiltakstype.toInnhold
+import no.nav.amt.lib.models.arrangor.melding.EndringFraArrangor
+import no.nav.amt.lib.models.deltaker.Deltakelsesinnhold
+import no.nav.amt.lib.models.deltakerliste.tiltakstype.Tiltakstype
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+const val TESTVEILEDER = "Z990098"
+const val TESTENHET = "0314"
+
+// Plutselig Lagsport
+const val TESTARRANGORANSATT = "fff9a665-cbde-4dbc-9ef9-deb8681a0d6f"
+
+class TestdataService(
+    private val pameldingService: PameldingService,
+    private val deltakerlisteService: DeltakerlisteService,
+    private val arrangorMeldingProducer: ArrangorMeldingProducer,
+    private val deltakerService: DeltakerService,
+) {
+    suspend fun opprettDeltakelse(opprettTestDeltakelseRequest: OpprettTestDeltakelseRequest): Deltaker {
+        val deltakerliste = deltakerlisteService.get(opprettTestDeltakelseRequest.deltakerlisteId).getOrThrow()
+        val forventetSluttdato = opprettTestDeltakelseRequest.startdato.plusMonths(3)
```

Comment:
Ville vurdert å ta inn også sluttdato i requesten her. Selv om det kanskje ikke er et konkret behov for dette akkurat nå så blir det mer fleksibelt og legger ikke egentlig til noe ekstra kompleksitet utenom at man da bør validere at sluttdato er etter startdato og at hvis startdato er null så må sluttdato også være det